openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -132,7 +132,7 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
                             min="0" max="99"
                         />
                     </div>
-                    <p class="setting-help">{% trans "Specify the number of top-scoring responses to display. If the number is 0, the Top Responses section won't appear to students." %}</p>
+                    <p class="setting-help">{% trans "Specify the number of top-scoring responses to display. If the number is 0, the Top Responses section won't appear to students. Maximum: 99" %}</p>
```

Comment:
@symbolist 
"Specify the number of top scoring responses to display. Valid numbers are 0 to 99. If the number is 0, the Top Responses section does not appear to learners."